Code/Editor/Core/EditorActionsHandler.cpp

```diff
@@ -158,6 +198,11 @@ EditorActionsHandler::~EditorActionsHandler()
         AzToolsFramework::EditorEntityContextNotificationBus::Handler::BusDisconnect();
         AzToolsFramework::EditorEventsBus::Handler::BusDisconnect();
         AzToolsFramework::ActionManagerRegistrationNotificationBus::Handler::BusDisconnect();
+
+        if (m_editorViewportDisplayInfoHandler)
+        {
+            delete m_editorViewportDisplayInfoHandler;
+        }
```

Comment:
nit: It's not necessary to check if a pointer is null before deleting it, deleting a nullptr is just a noop